spec/validations/translations_spec.rb

```
@@ -55,6 +55,32 @@
     end
   end
 
+  context 'when there is a en.json' do
+    context 'required keys are missing' do
+      let(:translation_files) do
+        [double('AppFile',
+                relative_path: 'translations/en.json',
+                read: read_fixture_file('invalid_en.json'),
+                to_s: 'translations/en.json')]
+      end
+
+      it 'should be invalid' do
```

nitpick: `should report the error`?